share/spice/namecheap/namecheap.js

```diff
@@ -0,0 +1,54 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_namecheap = function(api_result){
+        // turns on/off debugging output
+        var is_debug = false;
+
+        if (!api_result) {
+            return Spice.failed('namecheap');
+        }
+
+        var item = api_result.ApiResponse.CommandResponse.DomainCheckResult;
+
+
+        if (!item) {
+            return Spice.failed('namecheap');
+        }
+
+        /* extract data from JSON */
+        var available = item.Available;
+        var domainName = item.Domain;
+
+        if ( !available || !domainName ) {
+            return Spice.failed('namecheap');
+        }
+
+        var data = {
+            domainAvailable: available === "true",
+            domainName: domainName
+        };
+
+        Spice.add({
+            id               : 'namecheap',
+            name             : 'Namecheap',
```

Comment:
We're trying to generalize the tab names and prefer not to show any brand names, (e.g. Amazon tab is called "products", Soundcloud is "music") so let's change this to `Domain`